glib/src/commonMain/kotlin/io/github/cgr/glib/number/Dec.kt

```diff
@@ -0,0 +1,30 @@
+package io.github.cgr.glib.number
+
+expect class Dec {
+    fun abs(): Dec
+}
+
+expect fun newDec(value: String): Dec
+
+fun Int.toDec(): Dec = newDec(this.toString())
+fun Long.toDec(): Dec = newDec(this.toString())
+fun Double.toDec(): Dec = newDec(this.toString())
```

Comment:
Do these have to go through a String? I'd understand if the JS one does, but Java supports `new BigDecimal(double x)` and `new BigDecimal(long x)`.

Reply:
Yes, because the double could lose precision. Also it'd be tedious to make one specially for JVM.